### PR TITLE
Fix(DB/Quest): Remove Nipsy Handin location

### PR DESCRIPTION
**Pull request by Ollie-Armitage**
The quest location is showing in an incorrect place. As the handin is in deeprun tram, the quest location has been removed.

## Changes Proposed:
-  Removed quest location from the quest_poi and quest_poi_points tables, as it is in the deeprun tram.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9086

## Tests Performed:

-  Tested in game by checking the map on Windows. 

## How to Test the Changes:

1. Get an alliance character at level 10
2. .q add 6662
3. Select quest on log and click on "Show Map" or simply Open map to see where POI for quest is located.
4. Where previously it would show the quest as being in silverpine forest, it should no longer show a quest marker at all. 
5. EDIT:: It should now show the quest as being in the deeprun tram entrance in either stormwind or ironforge, depending on whichever is closest.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1636703045917932956.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1636703045917932956');
+
+DELETE FROM quest_poi_points WHERE QuestID = 6662;
+DELETE FROM quest_poi WHERE QuestID = 6662;
```
Comment (IntelligentQuantum) [inline data/sql/updates/pending_db_world/rev_1636703045917932956.sql:4]:
```suggestion
DELETE FROM `quest_poi_points` WHERE `QuestID` = 6662;
DELETE FROM `quest_poi` WHERE `QuestID` = 6662;
```

**Review by IntelligentQuantum — CHANGES_REQUESTED** [review]

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by Kitzunu** [comment]
Or move it to deeprun tram?

**Comment by Kitzunu** [comment]
Also there should be a POI on the entrance of deeprun tram

**Comment by FrancescoBorzi** [comment]
what's the status of this?

also @IntelligentQuantum don't forget to update the workflow labels

**Comment by Ollie-Armitage** [comment]
There's no deeprun tram map pre-cata, but additionally the script now includes inserting new poi points on stormwind / ironforge deeprun tram gates.

**Comment by savowe** [comment]
Tested:

 - [x] Get an alliance character at level 10
 - [x] .q add 6662
 - [x] Select quest on log and click on "Show Map" or simply Open map to see where POI for quest is located.
 - [x] Where previously it would show the quest as being in silverpine forest, it should no longer show a quest marker at all.
 - [x]  It shows quest markers on the entrances to deeptram in sw and if. 
 
It does not, as @Ollie-Armitage writes, in step 5 of 'how to test' ("EDIT:: It should now show the quest as being in the deeprun tram entrance in either stormwind or ironforge depending on whichever is closest."), show the quest marker depending on your position. One can always see both. But I guess this is expected.

questmarker position:



2021-11-19 21_41_43-World of Warcraft

empty silverpine forest:
2021-11-19 21_46_41-World of Warcraft

**Comment by Ollie-Armitage** [comment]
@savowe apologies when I wrote 'depending on whichever is closest' I meant it will select whichever is closest when selecting the show on map from the quest log. I believe being able to see both if looking at the relevant area is intended (?)

**Comment by savowe** [comment]
> @savowe apologies when I wrote 'depending on whichever is closest' I meant it will select whichever is closest when selecting the show on map from the quest log. I believe being able to see both if looking at the relevant area is intended (?)

Yes, right. I also thought it might be just poorly worded. Just wantet to make sure. And yes, I also think it just works as intended.

**Pull request merged**